deploy/jbossas/modeshape-jbossas-subsystem/src/main/resources/schema/modeshape_3_0.xsd

```diff
@@ -574,7 +583,25 @@
             </xs:annotation>
         </xs:attribute>
     </xs:complexType>
-  
+
+    <xs:complexType name="file-persistence">
+        <xs:attribute name="path" type="xs:string" use="optional" default="${jboss.server.data.dir}/modeshape">
+            <xs:annotation>
+                <xs:documentation>
+                    The absolute path on the FS where the repository data should be persisted. If not specified,
+                    defaults to ${jboss.server.data.dir}/modeshape/${repository.name}
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
```

Comment:
i never understood where that idiom came from in the first place and it always seemed over complicated to me since AS has environment variables for all the server-specific folders :wink: 

if there's some sort of logical explanation for separating paths in 2 I'm happy to change it